lib/recurly/account.php

```diff
@@ -13,7 +13,7 @@
  * @property string $state The state of accounts to return: active or closed.
  * @property string $username The username of the account.
  * @property string $email The email address of the account.
- * @property string[] $cc_emails Additional email address that should receive account correspondence. These should be separated only by commas. These CC emails will receive all emails that the email field also receives.
```

Comment:
This also seems wrong in the documentation https://dev.recurly.com/docs/account-object. By adding entries via the Recurly UI the API returns a comma separated list of emails

Reply:
Good catch. I'll update docs once this is merged.